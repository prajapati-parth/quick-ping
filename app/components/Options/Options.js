import React, {Component} from 'react'
import {
	Text,
	View,
    TouchableOpacity
} from 'react-native'
import CompStyles from './comp_styles'

export default class Options extends Component {
    constructor(props){
        super(props);
    }

    render() {
        return (
            <TouchableOpacity 
                style={CompStyles.optionCard}
                activeOpacity={0.7}
                onPress={this._onOptionPressed}
            >
                <Text>{this.props.name}</Text>
            </TouchableOpacity>
        )
    }

    _onOptionPressed() {

    }
}