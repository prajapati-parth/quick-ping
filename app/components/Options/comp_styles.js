import {StyleSheet} from 'react-native'
import GlobStyles from '../../config/styles'

const CompStyles = StyleSheet.create({
    optionCard: {
        backgroundColor: '#FFFFFF',
        flex: 1,
        elevation: GlobStyles.elevationForShadow,
        paddingTop: GlobStyles.boundaryPadding*1.5,
        paddingBottom: GlobStyles.boundaryPadding*1.5,
        paddingLeft: GlobStyles.boundaryPadding,
        paddingRight: GlobStyles.boundaryPadding,
        borderRadius: 3
    }
})

export default CompStyles