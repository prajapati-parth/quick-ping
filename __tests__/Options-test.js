import 'react-native'
import React from 'react'
import Options from '../app/components/Options'

import renderer from 'react-test-renderer'

test('renders correctly', () => {
    const tree = renderer.create(
        <Options />
    ).toJSON()
    expect(tree).toMatchSnapshot()
})