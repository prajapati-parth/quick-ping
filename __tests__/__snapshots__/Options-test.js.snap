// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "borderRadius": 3,
      "elevation": 5,
      "flex": 1,
      "opacity": 1,
      "paddingBottom": 15,
      "paddingLeft": 10,
      "paddingRight": 10,
      "paddingTop": 15,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  />
</View>
`;
